### PR TITLE
Different types of internet connections

### PR DESCRIPTION
**Pull request by carinadiesel-dev**
Different types of internet connections

Part of #25

### CODE REVIEW

File: content/topics/types-of-connections/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+title: Types of internet connections
+keywords: internet connections, types of internet connections, internet connections explained
+description: A collection of informational pieces that explains different types of internet connection.
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:4]:
```suggestion
description: A collection of informational pieces that explain different types of internet connections.
```

File: content/topics/types-of-connections/index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  main:
+    parent: topics
+    name: Types of internet connections
+---
+
+# Dial-up
+
+## What is a dial-up connection?
+
+A dial-up connection is established when two or more communication devices use a public switched telephone network (PSTN) to connect to an Internet service provider (ISP) at data transfer rates (DTR) of 56kbps, according to [Techopedia](https://www.techopedia.com/definition/25953/dial-up-connection)
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:15]:
```suggestion
According to [Techopedia](https://www.techopedia.com/definition/25953/dial-up-connection), a dial-up connection is established when two or more communication devices use a public switched telephone network (PSTN) to connect to an Internet Service Provider (ISP) at Data Transfer Rates (DTR) of 56kbps, 
```

File: content/topics/types-of-connections/index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+This means your computer connects to an active phone line and dials a number in order to connect to your ISP via an analog modem.
+
+The phone line cannot be in use during the connection and when making the connection.
+
+!(https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.tech-faq.com%2Fdial-up-connection.html&psig=AOvVaw1aJbdUbuhccAm5R9-EBUw_&ust=1665823821490000&source=images&cd=vfe&ved=0CA0QjRxqFwoTCIjBpLur3_oCFQAAAAAdAAAAABAE)
+
+_Listen to the sound of a dial up connection_
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:23]:
```suggestion
_Listen to the sound of a dial-up connection_
```

File: content/topics/types-of-connections/index.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+{{< youtube id="gsNaR6FRuO0" title="Listen to the sound of a dial up connection" >}}
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:25]:
```suggestion
{{< youtube id="gsNaR6FRuO0" title="Listen to the sound of a dial-up connection." >}}
```

File: content/topics/types-of-connections/index.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+### More Resources Info
+
+[Computer Hope](https://www.computerhope.com/jargon/d/dialup.htm)
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:29]:
```suggestion
### More Resources

[Computer Hope: Dial-up](https://www.computerhope.com/jargon/d/dialup.htm)
```

File: content/topics/types-of-connections/index.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+# Broadband
+
+## What is broadband connection?
+
+Broadband refers to high-speed Internet access that is always on and faster than traditional dial-up access. According to the [FCC](https://www.fcc.gov/general/types-broadband-connections), broadband internet should be capable of a minimum of 25 Mbps download and 3 Mbps upload speeds
+
+[Understanding Broadband Internet Speeds](https://www.lifewire.com/broadband-internet-speeds-explained-437202)
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:37]:
```suggestion
### More info

[Lifewire: Understanding Broadband Internet Speeds](https://www.lifewire.com/broadband-internet-speeds-explained-437202)
```

File: content/topics/types-of-connections/index.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+## Broadband includes several types of high-speed internet technologies such as:
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:39]:
```suggestion

Broadband includes several types of high-speed internet technologies such as:
```

File: content/topics/types-of-connections/index.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+# Fiber
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:41]:
```suggestion
## Fiber
```

File: content/topics/types-of-connections/index.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+{{< youtube id="E3DEJ7odWq0" title="fiber optic cables (what you NEED to know)" >}}
+
+According to [Enterprise Spectrum](https://enterprise.spectrum.com/support/faq/internet/how-does-fiber-internet-work.html) Fiber Internet provides Internet access using fiber-optic cable rather than traditional copper cable or satellite signals. Fiber-optic lines consist of up to hundreds of small strands of glass or plastic cables, each about 1/10th the size of a single human hair. Each strand of fiber cable transmits data using pulses of light that travel at speeds approaching the speed of light.
+
+_Still curious about fibre ? Read more_ [here](https://www.explainthatstuff.com/fiberoptics.html)
+
+# DSL
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:49]:
```suggestion
## DSL
```

File: content/topics/types-of-connections/index.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+DSL (Digital Subscriber Line) operates over standard copper telephone lines like dial-up service but is many times faster than dial-up. In addition, unlike dial-up, DSL does not tie up the phone line. Coexisting with telephone service in this way allows users to surf the Net and use the phone at the same time, according to [Easy Tech Junkie](https://www.easytechjunkie.com/what-is-dsl.htm)
+
+!(https://media.geeksforgeeks.org/wp-content/uploads/Capture-86.png)
+
+There are two types of DSL,namely Assymetric DSL (ADSL) and Symmetric DSL (SDSL).
+
+_To learn more about DSL,_ [click here](https://www.lifewire.com/different-types-of-dsl-technology-817522)
+
+# Cable
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:61]:
```suggestion
## Cable
```

File: content/topics/types-of-connections/index.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+{{< youtube id="IOZjdDHYB0g" title="How cable internet works" >}}
+
+[Techslang](https://www.techslang.com/what-is-cable-internet-and-how-does-it-work/) explains that Cable Internet is a form of high-speed connection that uses a cable television infrastructure to provide access to end users. Therefore, the network that provides your cable TV connection is the same one that allows you to access the Internet.
+
+!(https://www.upwardbroadband.com/wp-content/uploads/2020/01/How-Cable-Internet-Works.png)
+
+# Satellite
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:69]:
```suggestion
## Satellite
```

File: content/topics/types-of-connections/index.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+
+{{< youtube id="2cZB8E6lh8Y" title="How satellite internet works" >}}
+
+Unlike land-based internet services like DSL, fibre or cable which received its signal from cables either below or slightly above the ground, satellite internet is a wireless connection beamed down from satellites orbiting the earth.
+
+!(https://www.groundcontrol.com/en/wp-content/uploads/2021/12/How-data-moves-through-satellite-network.jpg)
+
+# Wireless
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:77]:
```suggestion
## Wireless
```

File: content/topics/types-of-connections/index.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+
+[https://www.youtube.com/watch?v=hePLDVbULZc&ab_channel=BRIGHTSIDE]
+{{< youtube id="hePLDVbULZc" title="How Wi-fi works" >}}
+
+According to [Verizon](https://www.verizon.com/articles/internet-essentials/broadband-definition/),Wireless broadband (Wi-Fi) connects a home or business to the internet using radio signals instead of cables.
+
+_Signal boosters go into much more details regarding wifi standards,types of wifi and much more_ [here](https://www.signalboosters.com/blog/what-is-wifi-and-how-does-it-work/)
+
+# Cellular
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:86]:
```suggestion
## Cellular
```

File: content/topics/types-of-connections/index.md [hunk 14]
```diff
@@ -0,0 +1,104 @@
...
+
+## (EDGE, 3G, H+, 4G, 5G)
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:88]:
```suggestion
### (EDGE, 3G, H+, 4G, 5G)
```

File: content/topics/types-of-connections/index.md [hunk 15]
```diff
@@ -0,0 +1,104 @@
...
+
+## Broadband includes several types of high-speed internet technologies such as:
+
+# Fiber
+
+{{< youtube id="E3DEJ7odWq0" title="fiber optic cables (what you NEED to know)" >}}
+
+According to [Enterprise Spectrum](https://enterprise.spectrum.com/support/faq/internet/how-does-fiber-internet-work.html) Fiber Internet provides Internet access using fiber-optic cable rather than traditional copper cable or satellite signals. Fiber-optic lines consist of up to hundreds of small strands of glass or plastic cables, each about 1/10th the size of a single human hair. Each strand of fiber cable transmits data using pulses of light that travel at speeds approaching the speed of light.
+
+_Still curious about fibre ? Read more_ [here](https://www.explainthatstuff.com/fiberoptics.html)
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:47]:
```suggestion
_Still curious about fiber? [Read more about it here](https://www.explainthatstuff.com/fiberoptics.html)_
```

File: content/topics/types-of-connections/index.md [hunk 16]
```diff
@@ -0,0 +1,104 @@
...
+
+# DSL
+
+{{< youtube id="mI6Uh6D3NvA" title="What is DSL" >}}
+
+DSL (Digital Subscriber Line) operates over standard copper telephone lines like dial-up service but is many times faster than dial-up. In addition, unlike dial-up, DSL does not tie up the phone line. Coexisting with telephone service in this way allows users to surf the Net and use the phone at the same time, according to [Easy Tech Junkie](https://www.easytechjunkie.com/what-is-dsl.htm)
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:53]:
```suggestion
According to [Easy Tech Junkie](https://www.easytechjunkie.com/what-is-dsl.htm), DSL (Digital Subscriber Line) operates over standard copper telephone lines like dial-up service but is many times faster than dial-up. In addition, unlike dial-up, DSL does not tie up the phone line. Coexisting with telephone service in this way allows users to surf the Net and use the phone at the same time.
```

File: content/topics/types-of-connections/index.md [hunk 17]
```diff
@@ -0,0 +1,104 @@
...
+
+!(https://media.geeksforgeeks.org/wp-content/uploads/Capture-86.png)
+
+There are two types of DSL,namely Assymetric DSL (ADSL) and Symmetric DSL (SDSL).
+
+_To learn more about DSL,_ [click here](https://www.lifewire.com/different-types-of-dsl-technology-817522)
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:59]:
```suggestion
There are two types of DSL, namely Asymmetric DSL (ADSL) and Symmetric DSL (SDSL). _Learn more about the [different types of DSL technology here](https://www.lifewire.com/different-types-of-dsl-technology-817522)_.
```

File: content/topics/types-of-connections/index.md [hunk 18]
```diff
@@ -0,0 +1,104 @@
...
+
+[Techslang](https://www.techslang.com/what-is-cable-internet-and-how-does-it-work/) explains that Cable Internet is a form of high-speed connection that uses a cable television infrastructure to provide access to end users. Therefore, the network that provides your cable TV connection is the same one that allows you to access the Internet.
+
+!(https://www.upwardbroadband.com/wp-content/uploads/2020/01/How-Cable-Internet-Works.png)
+
+# Satellite
+
+{{< youtube id="2cZB8E6lh8Y" title="How satellite internet works" >}}
+
+Unlike land-based internet services like DSL, fibre or cable which received its signal from cables either below or slightly above the ground, satellite internet is a wireless connection beamed down from satellites orbiting the earth.
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:73]:
```suggestion
Unlike land-based internet services like DSL, fiber or cable, which receive its signal from cables either below or slightly above the ground, satellite internet is a wireless connection beamed down from satellites orbiting the earth.
```

File: content/topics/types-of-connections/index.md [hunk 19]
```diff
@@ -0,0 +1,104 @@
...
+!(https://www.groundcontrol.com/en/wp-content/uploads/2021/12/How-data-moves-through-satellite-network.jpg)
+
+# Wireless
+
+[https://www.youtube.com/watch?v=hePLDVbULZc&ab_channel=BRIGHTSIDE]
+{{< youtube id="hePLDVbULZc" title="How Wi-fi works" >}}
+
+According to [Verizon](https://www.verizon.com/articles/internet-essentials/broadband-definition/),Wireless broadband (Wi-Fi) connects a home or business to the internet using radio signals instead of cables.
+
+_Signal boosters go into much more details regarding wifi standards,types of wifi and much more_ [here](https://www.signalboosters.com/blog/what-is-wifi-and-how-does-it-work/)
```
Comment (bpires) [inline content/topics/types-of-connections/index.md:84]:
```suggestion
_Understand what WiFi is and how it really works in [this article by Signal boosters](https://www.signalboosters.com/blog/what-is-wifi-and-how-does-it-work/)_.
```

**Review by bpires — CHANGES_REQUESTED** [review]
Hey @carinadiesel-dev, congrats on your write-up on such a broad subject! 🎉 

I went on and already done a proofread, fixed some typos and style issues. Please re-read everything and change what you think it's necessary. 

Also, I reorganized the heading levels. Please note that Markdown/HTML heading best practices requests that we nest different heading levels hierarchically.

> Headings with an equal or higher rank start a new section, headings with a lower rank start new subsections that are part of the higher ranked section. Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a `<h2>` is not followed directly by an `<h4>`, for example. 

Regarding the content, it seems good to me, but I have some concerns about references and originality.

I have to say I'm not a fan of using "according to.." so frequently, and, when using it, please note that, if you are mentioning the source text verbatim, you still have to put it between quotes. In that note, I think it's better to paraphrase the content, and the cite the sources as "references". Let me know what you think about it.

IMO, the best approach is to do a quick definition and then point to resources, like articles or videos, for more in-depth knowledge. When working with definitions sites like Wikipedia and alike are a good point to start. I like those references for the main content because Wikipedia (or MDN, for instance) is more unlikely to get outdated, be discontinued or go offline. For the "more resources", we can link to articles scattered on the internet and then, if they went offline, we will fix it.

Also, take a look at the images issues I pointed out. You should consider if it's really relevant to embed it. If so, fix the syntax and always reference the source. 

There's also an image on the repo (`speed-chart.png"`) that, unless I'm mistaken, it's not referenced anywhere in the text. Please take a look into it!

Remember, to continue working on this PR, just add new commits by pushing to the `25-how-is-data-sent-over-the-internet` branch on your fork.

**Comment by schalkneethling** [comment]
Wow, thank you for the detailed review, @bpires 🙌

**Comment by schalkneethling** [comment]
@carinadiesel-dev let me know when you have had a moment to address the review comments. I would be happy to do a second round of review but, I do not want to overwhelm you. Thanks!

**Comment by schalkneethling** [comment]
@carinadiesel-dev I believe this https://github.com/The-Mycelium-Network/webworkbook/pull/74 is a duplicate? Have a look at my comments there and apply them here. At that point, I would suggest closing the other one and keeping this one. Thanks!

**Comment by bpires** [comment]
> @carinadiesel-dev I believe this #74 is a duplicate? Have a look at my comments there and apply them here. At that point, I would suggest closing the other one and keeping this one. Thanks!

I think this one got mixed up with #74, which should be **just** about HTTP and HTTPS, but had the part of "types-of-connections" as well (the subject of this PR). Probably because @carinadiesel-dev did #74 directly on the main branch of her fork. Maybe we should draft up a step-by-step of the workflow when working in different PRs, because it gets confusing sometimes.

**Comment by schalkneethling** [comment]
Ping 🔔  @carinadiesel-dev when you have a moment. Thanks!

**Comment by schalkneethling** [comment]
@carinadiesel-dev we are going to merge this one and then iterate on improving and cleaning up. Thanks!

**Pull request merged**